packages/@glimmer/syntax/test/public-api-test.ts

```diff
@@ -0,0 +1,37 @@
+import * as syntax from '@glimmer/syntax';
+
+QUnit.module('[glimmer-syntax] Public API is unchanged');
+
+QUnit.test('exports are not accidentally removed', function(assert) {
```

Comment:
if there is a better way to do this, I'd be all ears. I think it'd be great to have some sort of snapshot tool to at least automatically manage `Object.keys(syntax)`